insights/parsers/ironic_conf.py

```diff
@@ -0,0 +1,56 @@
+
+"""
+IronicConf - file ``/etc/ironic/ironic.conf``
+=============================================
+This class provides parsing for the file ``/etc/ironic/ironic.conf``.
+
+Sample input data is in the format::
```

Comment:
@smalleni you need to include the file `docs/shared_parsers_catalog/ironic_conf.rst` so that your parser docs will be rendered.  You'll also need to test your docs as they have some formatting issues, for instance there should be a blank line after line 7 in order to properly format the **preformatted** text.  See the [documentation guidelines](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html) for more information.

A sample `ironic_conf.rst` looks like this:

```
.. automodule:: insights.parsers.ironic_conf
   :members:
   :show-inheritance:
```